src/Test/Utilities/Shared/Metadata/ILBuilderVisualizer.cs

```diff
@@ -70,7 +69,7 @@ private static List<HandlerSpan> GetHandlerSpans(ImmutableArray<Cci.ExceptionHan
             {
                 HandlerSpan span;
 
-                if (region.HandlerKind == ExceptionRegionKind.Filter)
+                if (region.HandlerKind == System.Reflection.Metadata.ExceptionRegionKind.Filter)
```

Comment:
`ILOpCode` (used elsewhere in this file) is defined in two namespaces, one of which is `System.Reflection.Metadata.ILOpCode`, so I removed `using System.Reflection.Metadata;`.